tests/components/demo/test_remote.py

```diff
@@ -3,13 +3,20 @@
 import unittest
 
 import homeassistant.components.remote as remote
-from homeassistant.const import STATE_OFF, STATE_ON
+from homeassistant.components.remote import ATTR_COMMAND
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_TURN_OFF,
+    SERVICE_TURN_ON,
+    STATE_OFF,
+    STATE_ON,
+)
 from homeassistant.setup import setup_component
 
 from tests.common import get_test_home_assistant
-from tests.components.remote import common
 
 ENTITY_ID = "remote.remote_one"
+SERVICE_SEND_COMMAND = "send_command"
 
 
 class TestDemoRemote(unittest.TestCase):
```

Comment:
This class will also need to be rewritten to use PyTest style.

Reply:
Did that with #41845.